scripts/start

```diff
@@ -24,6 +34,22 @@ function normalizePort(val) {
   return false;
 }
 
+function onShutdown() {
+  console.log("\x1b[32m");
+  console.info('Got SIGTERM. Graceful shutdown start', new Date().toISOString())
+  server.close(async (err) => {
+    if (err) {
+      console.poo(" "+err)
+      process.exit(1)
+    } else {
+      console.tick(" Server health restored");
```

Comment:
The extra whitespace here is not required:
```suggestion
      console.tick("Server health restored");
```
Since you're already appending a whitespace after the emoji in line 11.